src/category_theory/sums/associator.lean

```diff
@@ -5,7 +5,7 @@ Authors: Scott Morrison
 -/
 import category_theory.sums.basic
 
-/-#
+/-!
```

Comment:
Should there be a header to this module doc?

Reply:
Done!